feat: default_sort_order (#838)
nextcord/nextcord: docs/api.rst

            This action type can only be used with the :attr:`Permissions.moderate_members` permission.

> .. class:: SortOrderType

Reviewer: ```suggestion
.. class:: SortOrderType

    .. versionadded:: 2.3
```

>     The default sort order type used to sort posts in forum channel.

Reviewer: ```suggestion
    The default sort order type used to sort posts in a :class:`ForumChannel`.
```

            This action type can only be used with the :attr:`Permissions.moderate_members` permission.

.. class:: SortOrderType

    .. versionadded:: 2.3

    The default sort order type used to sort posts in a :class:`ForumChannel`.

    .. attribute:: latest_activity

>         Sort forum posts by its activity.

Reviewer 2: ```suggestion
        Sort forum posts by their activity.
```

    .. attribute:: creation_date

>         Sort forum posts by its creation date.

Reviewer 2: ```suggestion
        Sort forum posts by their creation date.
```
